src/components/modals/SharedLinkSettingsModal/SharedLinkSettingsModal.tsx

```diff
@@ -138,12 +138,12 @@ export const SharedLinkSettingsModal: React.FC<SharedLinkSettingsModalProps> = (
 
     // If we already have a generated share link, copy it
     if (generatedShareLink) {
-      Clipboard.setString(generatedShareLink);
+      await Clipboard.setStringAsync(generatedShareLink);
```

Comment:
Fixes some deprecation warnings since the other function blocks the thread for a moment since is synchronous and native (very bad idea)

Reply:
Nice